### PR TITLE
[DeadCode]  Skip RemoveUnusedConstructorParamRector on property promotion

### LINKED ISSUE #5533: [RemoveUnusedConstructorParamRector] Broken behaviour with php8 constructor property promotion and empty body
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.9.27 |
| Installed as   | composer dependency                          |

```diff
--- Original
+++ New
@@ @@
      */
     private const TABLE_NAME = 'storage';

-    public function __construct(
-        private Connection $connection
-    ) {
+    public function __construct()
+    {
     }

     public function add(UuidInterface $id): void
```

https://getrector.org/demo/d4666d60-65d4-4fac-9b0f-fdb50601a84a

## Expected Behaviour

Constructor property promotion should be present after fixing because it used in methods

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes #5533

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by dvapelnik** [comment]
@samsonasik thansk a lot!

File: rules/symfony/src/Rector/Class_/MakeCommandLazyRector.php [hunk 1]
```diff
@@ -199,20 +210,6 @@ private function removeConstructorIfHasOnlySetNameMethodCall(Class_ $class): voi
...
-        foreach ($params as $param) {
-            if ($param->flags !== 0) {
-                return true;
-            }
-        }
-
-        return false;
-    }
-
     private function matchCommandNameNodeInConstruct(StaticCall $staticCall): ?Expr
```
Comment (TomasVotruba) [inline rules/symfony/src/Rector/Class_/MakeCommandLazyRector.php:213]: :+1:

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**